test/integration/running/test_running.py

```diff
@@ -568,6 +576,9 @@ def test_running_env_variables(tt_cmd, tmpdir_with_cfg):
     # Check that log format is in json.
     isJson = False
     logPath = os.path.join(tmpdir, "var", "log", "test_env_app", "test_env_app.log")
+    logDir = os.path.join(tmpdir, "var", "log", "test_env_app")
```

Comment:
The process has been correctly terminated, but the log file was not created?

Reply:
Yes :), I have no clue how it is even possible and couldn't reproduce it even under heavy stress test on my working machine.